tensorflow/python/kernel_tests/stack_op_test.py

```diff
@@ -49,10 +49,10 @@ def randn(self, shape, dtype):
     else:
       return data.astype(dtype)
 
-  @test_util.run_deprecated_v1
```

Comment:
Please remove test_util.run_deprecated_v1 so that we are exercising eager mode. Same applies to all the other changes in this file.